Resources/doc/reference/installation.rst

```diff
@@ -107,3 +107,10 @@ Installation
                 # ... other blocks
 
                 - { position: center, type: sonata.timeline.block.timeline, settings: { context: SONATA_ADMIN, max_per_page: 25 }}
+
+
+* Run the Doctrine update command:
+
+.. code-block:: bash
+    
+    php app/console doctrine:schema:update --force
```

Comment:
Can link to: http://symfony.com/doc/current/book/doctrine.html#creating-the-database-tables-schema
> generates the SQL statements needed to update the database to where it should be

``--dump-sql`` could be used to see what will be done.

Or add the migrations commands:
```bash
php app/console doctrine:migrations:diff
php app/console doctrine:migrations:migrate
```